src/components/lime-ai-avatar/lime-ai-avatar.tsx

```diff
@@ -0,0 +1,61 @@
+import { Component, h, Host, Prop } from '@stencil/core';
+
+/**
+ * This component displays an avatar, representing Lime AI assistants.
+ * :::warning
+ * This is a private component used internally in the Lime's various applications,
+ * which is the reason for having it in Lime Elements —to ease the distribution
+ * of the component across all our apps.
+ *
+ * 3rd party developers are not allowed use this component directly.
+ * :::
+ *
+ * @private
+ * @exampleComponent limel-example-lime-ai-avatar-basic
+ * @exampleComponent limel-example-lime-ai-avatar-colors
+ */
+@Component({
+    tag: 'limel-lime-ai-avatar',
```

Comment:
It seems obvious that this component is meant to be used in the card header, but is the intention that it's also used beside the bot's messages in the chat list? 👇

![image](https://github.com/user-attachments/assets/dfc887f5-bf66-41dc-896c-15ff2e88bc5c)

If it's not meant to be used beside the messages, then I'm not sure "avatar" should be in the name of this component, because to me, the avatar is the little image next to the chat messages, so it becomes confusing if this isn't meant to be used for that 🙂

Reply:
we will reuse it there (for bot's messages) as well

Reply:
https://github.com/Lundalogik/crm-client/issues/37